config/locales/forms/2023/lettings/income_and_benefits.en.yml

```diff
@@ -0,0 +1,183 @@
+            question_text: "Does the household pay rent or other charges for the accommodation?"
+
+          period:
+            page_header: ""
+            check_answer_label: "Frequency of household rent and charges"
+            hint_text: "Select how often the household is charged. This may be different to how often they pay."
+            question_text: "How often does the household pay rent and other charges?"
+
+          care_home:
+            page_header: ""
+            is_carehome:
+              check_answer_label: "Care home accommodation"
+              hint_text: ""
+              question_text: "Is this accommodation a care home?"
+            chcharge_weekly:
+              check_answer_label: "Care home charges"
+              hint_text: ""
+              question_text: "How much does the household pay every week?"
+            chcharge_bi_weekly:
+              check_answer_label: "Care home charges"
+              hint_text: ""
+              question_text: "How much does the household pay every 2 weeks?"
+            chcharge_4_weekly:
+              check_answer_label: "Care home charges"
+              hint_text: ""
+              question_text: "How much does the household pay every 4 weeks?"
+            chcharge_monthly:
+              check_answer_label: "Care home charges"
+              hint_text: ""
+              question_text: "How much does the household pay every month?"
+
+            rent_weekly:
+              page_header: "Household rent and charges"
+              brent:
+                check_answer_label: "Basic rent"
+                hint_text: "This is the amount paid before any charges are added for services (for example, hot water or cleaning). Households may receive housing benefit or Universal Credit towards basic rent."
+                question_text: "What is the basic rent?"
+              scharge:
+                check_answer_label: "Service charge"
+                hint_text: "For example, for cleaning. Households may receive housing benefit or Universal Credit towards the service charge."
+                question_text: "What is the service charge?"
+              pscharge:
+                check_answer_label: "Personal service charge"
+                hint_text: "For example, for heating or hot water. This doesn’t include housing benefit or Universal Credit."
+                question_text: "What is the personal service charge?"
+              supcharg:
+                check_answer_label: "Support charge"
+                hint_text: "Any charges made to fund support services included in tenancy agreement."
+                question_text: "What is the support charge?"
+              tcharge:
+                check_answer_label: "Household rent and charges"
+                hint_text: "This is the total for rent and all charges."
+                question_text: "Total charge"
```

Comment:
This is all repeated for rent_weekly, rent_bi_weekly, rent_4_weekly and rent_monthly. Is there any scenario where these questions will ever be different? Based of the rent period. Could they be merged into just a "rent" key?